src/cpp/dependencies.cmake

```diff
@@ -135,7 +135,6 @@ else()
 	)
 	FetchContent_GetProperties(pybind11)
 	if(NOT pybind11_POPULATED)
-		FetchContent_Populate(pybind11)
-		add_subdirectory(${pybind11_SOURCE_DIR} ${pybind11_BINARY_DIR})
```

Comment:
is it safe to remove this line aswell?

Reply:
Yes, `MakeAvailable` is more or less a shortcut for 
```
FetchContent_GetProperties(pybind11)
if(NOT pybind11_POPULATED)
  FetchContent_Populate(pybind11)
  add_subdirectory(${pybind11_SOURCE_DIR} ${pybind11_BINARY_DIR})
endif()
```

see docs for details: https://cmake.org/cmake/help/v3.27/module/FetchContent.html#overview